src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -2003,6 +2003,44 @@
      * The {@code continue} keyword.  The first child is an
      * optional identifier and the last child is a semicolon.
      *
+     * <p>For example:</p>
+     * <pre>
+     * for (int i = 0; i < 5; i++) {
```

Comment:
@MahmoudAbdelazim Pls use `&gt;` in place of `<` on this line. This will help pass the **drone CI test**.